schema/seed.manifest.schema.json

```diff
@@ -188,6 +191,42 @@
 									}
 								}
 							},
+							"env_vars": {
```

Comment:
I'm not overly picky on the name as a whole. I'd probably lean toward `envVars`, but that's more of a personal preference.